src/zc/form/browser/tests.py

```diff
@@ -63,24 +65,24 @@ def test_render(self):
         widget = UnionWidget(field, request)
         widget.setPrefix('field')
         output = widget()
-        self.failUnless('<table' in output)
-        self.failUnless('Age' in output)
-        self.failUnless('Name' in output)
-        self.failUnless(re.search(
+        self.assertTrue('<table' in output)
+        self.assertTrue('Age' in output)
+        self.assertTrue('Name' in output)
```

Comment:
It would be even nicer to use `self.assertIn(..., output)`.

Reply:
There are some other places later on in the file, too.

Reply:
I have changed that to more specific assertions. 
The is actually a reason to drop testing of Python 3.4:
```
 assertNotRegex = getattr(unittest.TestCase, 'assertNotRegex',
                          unittest.TestCase.assertNotRegexpMatches)
```
This does not work because `.assertNotRegexpMatches` is not yet marked as deprecated and leads to an `AttributeError`, although `.assertRegexpMatches` actually is available.